session/session.go

```diff
@@ -2644,22 +2633,26 @@ var builtinGlobalVariable = []string{
 
 var (
 	loadCommonGlobalVarsSQLOnce sync.Once
-	loadCommonGlobalVarsSQL     string
+	loadCommonGlobalVarsStmt    ast.StmtNode
```

Comment:
The DATA RACE is caused by here, in the test cases, multiple test case run `loadCommonGlobalVariablesIfNeeded` and they will use `astNode.Accept(vistor)` (which is not read-only operation) in statement rewriter.